screens/Home.tsx

```diff
@@ -1,19 +1,38 @@
-import React from 'react'
+import React from "react";
 import { Text, View } from "react-native";
-import HeaderBar from '~/components/HeaderBar';
-import Micro from '~/components/Micro';
+import HeaderBar from "~/components/HeaderBar";
+import TodoListItem from "~/components/TodoListItem";
 
-type Props = {}
+import Micro from "~/components/Micro";
+
+type Props = {};
 
 const Home = (props: Props) => {
   return (
-    <View className='w-full h-screen'>
-        <HeaderBar></HeaderBar>
-        <View className='bg-[#EEEEF0] h-full'>
-        </View>
-        <Micro></Micro>
+    <View className="w-full h-screen">
+      <HeaderBar></HeaderBar>
+      <View className="bg-[#EEEEF0] h-full">
```

Comment:
```suggestion
      <View className="bg-[#EEEEF0] h-full px-5">
```
组件布局由外部设置，组件本身不限制大小，可以支持更好的响应式布局